vhdbuilder/packer/components.json

```diff
@@ -67,6 +67,7 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
+        "v1.3.3-2",
```

Comment:
why it's not `v1.3.3.2`, is `v1.3.3-2` identical to `v1.3.3.2`?

Reply:
btw, I am going to update v1.3.4 release this week: https://github.com/kubernetes-sigs/secrets-store-csi-driver/releases/tag/v1.3.4, will overwrite `v1.3.3-2` directly

Reply:
here is the PR: https://github.com/Azure/AgentBaker/pull/3315

Reply:
* yes, they're identical, in the future we'll be using v1.3.3-2. "v1.3.3.1 is not compatible with semantic versioning libraries used by many tools (specifically Helm), whereas v1.3.3-1 is. ", according to Jeremy regarding this change.
* okay, will update corresponding PR too